.github/workflows/R-CMD-check.yaml

```
@@ -21,10 +25,17 @@ jobs:
       matrix:
         config:
           - {os: macos-latest,   r: 'release'}
+
           - {os: windows-latest, r: 'release'}
+          # use 4.0 or 4.1 to check with rtools40's older compiler
+          - {os: windows-latest, r: 'oldrel-4'}
```

```suggestion
          # use 4.0 or 4.1 to check with rtools40's older compiler
          - {os: windows-latest, r: 'oldrel-4'}
```

also failing because testing old version of R